policies/experimental-features.md

```diff
@@ -0,0 +1,52 @@
+Experimental Feature Policy
+===========================
+
+A feature in OpenSSL may be designated as an experimental feature.
+
+No stability or compatibility guarantees are offered for experimental features.
+An experimental feature, or any API, protocol or file format which is part of an
+experimental feature, may be changed arbitrarily and in compatibility-breaking
+ways, or removed entirely at any time, including between releases.
+
+An experimental feature is not enabled by default and must be explicitly enabled
+at build time in order to be used. The enable flag to enable an experimental
+feature must have the form `enable-experimental-XXX`. The corresponding `ifdef`
+shall have the form `OPENSSL_NO_EXPERIMENTAL_XXX`.
+
+A prominent warning should be issued by the configure script if any experimental
+feature is enabled, with a link to a webpage where people can learn more about
+experimental features.
+
+Experimental features may have associated documentation. This documentation is
+not built by default unless the corresponding experimental feature is enabled.
+Man pages for experimental features shall feature a prominent warning that the
+man page relates to an experimental feature.
+
+New internal header files for experimental features can be placed in `internal/`
+as usual. New public header files for experimental features should be named
+`<experimental/XXX.h>`. Such headers shall only be copied at install time if the
```

Comment:
This needs to be something like `<openssl-experimental/XXX.h>` or `<openssl/experimental/XXX.h>` to keep the convention of not invading other apps header file spaces.

Reply:
Oops, thought I wrote that...